docs/source/executing-operations/subscription-support.mdx

```diff
@@ -101,9 +101,11 @@ subscription:
         reviews: # Overrides settings for the 'reviews' subgraph
           path: /ws # Absolute path that overrides '/subscriptions' defined above
           protocol: graphql_ws # The WebSocket-based subprotocol to use for subscription communication (Default: graphql_ws)
+          heartbeat_interval: enable # Optional (Default: disabled)
```

Comment:
Could you add document to explain it also support human readable duration like `10secs`

Reply:
will do 👍

Reply:
fixed